apps/fem/fem2.cpp

```diff
@@ -0,0 +1,139 @@
+#include "graph.h"
```

Comment:
Rolls off the tongue! :D 

That sounds good, `triangle_scalar_2D.cpp` is pretty good. And +1 to omitting `fem`.